.docker/Pipfile

```diff
@@ -56,4 +56,4 @@ yaspin = "*"
 [dev-packages]
 
 [requires]
-python_version = "3.8"
+python_version = "3.8"
```

Comment:
Please revert changes to current main status for Pipfile and Pipfile.lock

Reply:
Can we just override these?

Reply:
Thanks for the tips @Debskij @raymondlo84 , I have overwritten these files and removed from this pull request.